apps/designer/app/shared/dom-hooks/use-scroll-state.ts

```diff
@@ -22,8 +22,10 @@ if (typeof window === "object") {
   let isScrolling = false;
 
   emitter.on("scroll", () => {
+    if (isScrolling === false) {
```

Comment:
still no guaranty that scrollStart will be earlier vs scroll ;) depends on emitter code